src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/GetActiveCollectionPeriod.cs

```diff
@@ -21,7 +21,7 @@ public GetActiveCollectionPeriod(IQueryDispatcher queryDispatcher, ILogger<GetAc
         [FunctionName(nameof(GetActiveCollectionPeriod))]
         public async Task<CollectionPeriod> Get([ActivityTrigger] object input)
         {
-            _logger.LogInformation("Getting active collection period");
+            _logger.LogDebug("Getting active collection period");
```

Comment:
Can implement the ILogWriter on the request/response objects to remove this logging from here.